managementApp/src/main/java/com/skytouch/service/ProductMessageSenderImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.skytouch.service;
+
+import com.skytouch.model.Product;
+import org.springframework.amqp.core.AmqpTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductMessageSenderImpl implements ProductMessageSender {
```

Comment:
The class should be default, that is because the service and the implementation are int the same package and the initialization of the service is created for the container (the default scope).